### ISSUE TITLE
Possible bugs in the atm/sfcship YAML?

### ISSUE THREAD

**Issue by danholdaway** [NOAA-EMC/GDASApp#1000]
When converting YAMLs to JCB I found these potential bugs in the sfcship YAMLs. There are several duplicate keys which means only one can be used (which one may depend on eckit policy). It looks like there should perhaps be a separate bounds/background checks for each variable rather than just one as there is presently. It's a Valid YAML so may cause confusion later. 

https://github.com/NOAA-EMC/GDASApp/blob/3ffc340fec25dcd53918194baac98226d01e1179/parm/atm/obs/config/sfcship.yaml.j2#L33

**Comment by ADCollard** [comment]
@danholdaway Just so you know, this is actually a placeholder.  This YAML need to be reviewed and replaced anyway.  But I can fix the YAML to at least make it unambiguous.

**Comment by danholdaway** [comment]
Fair enough, I wondered if that might be the case. You can also wait until JCB replaces it.

**Comment by danholdaway** [comment]
@ADCollard I also see that in the [ATMS N20 use_flag list](https://github.com/NOAA-EMC/GDASApp/blob/3ffc340fec25dcd53918194baac98226d01e1179/parm/atm/obs/config/atms_n20.yaml.j2#L527) there is 1s and a 4. Does 4 have a meaning different than 1?
